app/pages/lab-pages-editor/components/TasksPage/components/StepItem/StepItem.jsx

```diff
@@ -64,7 +71,27 @@ function StepItem({
     setActiveDragItem(stepIndex);  // Use state because DropTarget's onDragEnter CAN'T read dragEvent.dataTransfer.getData()
   }
 
-  const branchingTaskKey = canStepBranch(step, allTasks);
+  function experimentalRestyleContainer() {
+    const container = htmlContainer?.current;
+    const content = htmlContent?.current;
+    if (!content || !container || !ENABLE_EXPERIMENTAL_CONTAINER_STYLE) return;
+
+    const lastTaskInStep = taskKeys.at(-1);
+    const lastTaskHasAnswers = allTasks?.[lastTaskInStep]?.answers?.length > 0
+    const contentHeight = content.offsetHeight;
+    if (lastTaskInStep === branchingTaskKey && lastTaskHasAnswers && contentHeight) {
+      setContainerStyle({
+        height: `${content.offsetHeight - CONTAINER_HEIGHT_OFFSET}px`,
+        marginBottom: `${CONTAINER_HEIGHT_OFFSET}px`,
+        // overflow: 'visible'  // Unnecessary here, as this is already specified in the .styl
+      });
+    } else {
+      setContainerStyle({});
+    }
+  }
+
+  // TODO: experimentalRestyleContainer() should also trigger when the window resizes.
```

Comment:
When I resized my window, it looks like step containers resize as expected 🤔 Do you have screenshots of buggy behavior during resize?

Reply:
Sure do:

https://github.com/zooniverse/Panoptes-Front-End/assets/13952701/c1e315d0-c587-4d22-b681-552468c29fc2

Basically, the issue happens whenever the window resize is sufficient to trigger the row of answers to wrap-around.